src/main/java/dto/Photo.java

```diff
@@ -7,5 +7,5 @@
     private int photoId;
     private String path;
     private String fileName;
-    private String comments;
+    private String[] comments;
```

Comment:
Code Review input # 3 - Data type

A car could have many comments, so rather than a string, it should be an array of string.

This is shown on the official Lombok website, where they use "tags" as an example of an array of string:
https://projectlombok.org/features/Data

Reply:
Is this fully implemented in the code review?

This looks like the start of a new feature.  Avoid adding new features in a code review, as a good code review should merge to master, and only features that meet the team's Definition of Done should be merged to master.  If functionality is introduced and not completed, it adds to technical debt, as it's unused code.